src/Query.php

```diff
@@ -770,6 +770,10 @@ protected function _sub_render_condition($row)
         if (count($row) === 2) {
             $value = $cond;
 
+            if ($value instanceof Expressionable) {
```

Comment:
where it is needed? only for count() === 2?

Reply:
Yes, it is intended to uncover a nested Query and resolf correctly a few lines below in `instance of self`
In atk4/data (revamped) we will have `Persistence\Sql\Query implements Expressionable`. On getDsqlExpression it returns the underlying dsql\Query.
This change here is intended to uncover the dsql\Query so the operator is resolved correctly to 'in' and not '='

Reply:
A little cryptic - but if this will be covered by some test (in data, that is enought) - then ok

enforce return type of `dsql\Expression` from all `getDsqlExpression` methods